.github/ISSUE_TEMPLATE/report-bug.md

```
@@ -20,8 +26,37 @@ Please describe what's happening and what you would have expected instead.
+- Graphics hardware (`fastfetch --pipe` or `inxi -GSx`):
+
+## Logs
+
+Logs make almost every report faster to resolve. See
+[collecting logs](https://github.com/tonyantony300/dashbeam/blob/main/docs/troubleshooting.md#collecting-logs)
+for the command to use on your platform, then paste the output or attach the file.
+
+<details>
+<summary>dashbeam.log</summary>
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)